deploy.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd /home/ubuntu/build/codeit-resources
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)